DescendantBooks/po/it-local.po

```diff
@@ -0,0 +1,739 @@
+# Italian translation for Gramps DescendantBooks addon.
+#
+# This file is distributed under the same license as the Gramps package.
+#
+# Copyright (c) 2019 Giansalvo Gusinu <giansalvo.gusinu@gmail.com>
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: DescendantBooks Report addon for Gramps\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-01-22 09:28-0600\n"
+"PO-Revision-Date: 2019-02-06 11:25+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2.1\n"
+"Last-Translator: Giansalvo Gusinu <giansalvo.gusinu@gmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: it_IT\n"
+
+#: DescendantBooks/CollectAscendants.py:178
+#, python-format
+msgid "Deep pruning ascendants from %s people..."
+msgstr "Rimozione approfondita degli ascendenti in eccesso da %s persone..."
+
+#: DescendantBooks/CollectAscendants.py:182
+#, python-format
+msgid "Pruning ascendants from %s people..."
+msgstr "Rimozione degli ascendenti in eccesso da %s persone..."
+
+#: DescendantBooks/CollectAscendants.py:320
+#: DescendantBooks/CollectAscendants.py:413
+#, python-format
+msgid "Getting ascendants from %s people..."
+msgstr "Lettura degli ascendenti da %s persone..."
+
+#: DescendantBooks/CollectAscendants.py:331
+#, python-format
+msgid "Verifying descendants from %s people..."
+msgstr "Verifica dei discendenti da %s persone..."
+
+#: DescendantBooks/CollectAscendants.py:348
+#, python-format
+msgid "Getting missing descendants from %s people..."
+msgstr "Recupero dei discendenti mancanti da %s persone..."
+
+#: DescendantBooks/CollectAscendants.py:367
+#, python-format
+msgid "Verifying descendants from %s people after pruning..."
+msgstr "Verifica dei discendenti da %s persone dopo la rimozione..."
+
+#: DescendantBooks/CollectAscendants.py:383
+#, python-format
+msgid "Getting missing descendants from %s people ..."
+msgstr "Recupero dei discendenti mancanti da %s persone..."
+
+#: DescendantBooks/DescendantBookReport.py:218
+#: DescendantBooks/DescendantBookReport.py:223
+#, python-format
+msgid "sp. %(spouse)s"
+msgstr "sp. %(spouse)s"
+
+#: DescendantBooks/DescendantBookReport.py:233
+#, python-format
+msgid "sp. see  %(reference)s : %(spouse)s"
+msgstr "sp. vedi %(reference)s : %(spouse)s"
+
+#: DescendantBooks/DescendantBookReport.py:244
+#, python-format
+msgid "see report: %(report)s, ref: %(reference)s : %(person)s & %(spouse)s"
+msgstr ""
+"vedi resoconto: %(report)s, rif: %(reference)s : %(person)s & %(spouse)s"
+
+#: DescendantBooks/DescendantBookReport.py:319
+#, python-format
+msgid "%s sp."
+msgstr "%s sp."
+
+#: DescendantBooks/DescendantBookReport.py:369
+msgid "Descendants Report"
+msgstr "Resoconto dei Discendenti"
```

Comment:
"Resoconto dei discendenti" (not sure how much this differs from "Descendant Report" without the "s")
In general, please don't use The CamelCase Writing.

Reply:
ok